### PR TITLE
Fixes #23962: Adapt score UI to accept plugins 

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/23962

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/elm/sources/Node.elm [hunk 1]
```diff
@@ -25,11 +39,28 @@ update msg model =
...
+          processApiError "Getting score details" err model
+    ReceiveDetails name value ->
+        case Html.Parser.run value of
+          Ok htmlString ->
+            let
+               filterScript html =
+                   case html of
+                       Element "script" _ _ -> Text ""
+                       Element tag attributes children -> Element tag attributes (children |> List.filter filterScript)
+                       s-> s
```
Comment (clarktsiory) [inline webapp/sources/rudder/rudder-web/src/main/elm/sources/Node.elm:55]: Is that a guard against potential script elements passed in the html string ? 
The library seems to disallows scripts (like Elm, it does not have it in the Html tree representation : https://stackoverflow.com/a/74528143), and maybe this check should be removed then

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**